test_pull_requests

```diff
@@ -288,13 +289,15 @@ def extract_bot_comment_fields(bot_comment_body, settings=nil)
 
   build_url = bot_comment_body[build_url_pattern, 1]
   base_commit = bot_comment_body[/ \(Base Commit: ([a-f0-9]{40})\)/, 1]
+  pr_branch_commit = bot_comment_body[/ \(PR Branch Commit: ([a-f0-9]{40})\)/, 1]
```

Comment:
Is this just going to be `NULL` if there's no match? Will we ever have null-deref issues? Not clear on how Ruby will handle us indexing into this later...